<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainContainerLogout</class>
 <widget class="QWidget" name="mainContainerLogout">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">
#mainContainerLogout{
background-color:#272741;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel{
color:white;
font-size:14px;
}
QPushButton{
background-color:#1D1F32;
color:white;
font-size:14px;
border-radius:15px;
border:1px solid #678ADD;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>79</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Are you sure you want to Logout?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogout">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
